spec/webauthn/attestation_statement/android_safetynet_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+require "base64"
+require "jwt"
+require "openssl"
+require "webauthn/attestation_statement/android_safetynet"
+
+RSpec.describe "android-safetynet attestation" do
+  describe "#valid?" do
+    let(:statement) { WebAuthn::AttestationStatement::AndroidSafetynet.new("ver" => version, "response" => response) }
+    let(:version) { "2.0" }
+
+    let(:response) do
+      JWT.encode(
+        payload,
+        attestation_key,
+        "ES256",
```

Comment:
I believe Google has been using RS256 exclusively so far - switching to that would make for a slightly more realistic test. Guess we should tighten that up in the code as well.